### ISSUE TITLE
Is this the rawr CRAN package that was removed in 2021?

### ISSUE THREAD

**Issue by ReedMerrill** [raredd/rawr#1]
Hi, is this the package that was on CRAN until 2021? This is the current CRAN link: https://cran.r-project.org/web/packages/rawr/index.html

I'm asking because I was considering naming my own package "rawr". 

Thanks :blush: 

Reed

**Comment by raredd** [comment]
Nope, this package has never been on cran, so you have my blessing to use it :}
